web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -85,20 +87,18 @@
 </script>
 
 <IntersectionObserver once={false} on:intersected let:intersecting>
-  <!-- svelte-ignore a11y-no-noninteractive-tabindex -->
-  <div
+  <a
+    href={disabled ? '' : currentUrlReplaceAssetId(asset.id)}
```

Comment:
Only other change I would suggest is removing this ternary once again. With the `href` set to an empty string, the href becomes the current URL. Because `onIconClickedHandler` still handles the disabled state by stopping the default event behavior, we can simply remove it to allow Ctrl/Cmd+click or middle mouse click to take you to the proper URL.